README.md

```diff
@@ -18,7 +18,8 @@ There is no `latest` tag for this image
 - [helm-push](https://github.com/chartmuseum/helm-push) (latest commit)
 - [aws-iam-authenticator](https://github.com/kubernetes-sigs/aws-iam-authenticator) (latest version when run the build)
 - [eksctl](https://github.com/weaveworks/eksctl) (latest version when run the build)
-- [awscli v1](https://github.com/aws/aws-cli) (latest version when run the build)
+- [awscliv1](https://github.com/aws/aws-cli) (latest version when run the build)
+- [awscliv2](https://github.com/aws/aws-cli) (v2.1.39)
```

Comment:
can we always install the latest version for aws cli v2 as well?

Reply:
In this case, you will receive the following stacktrace:
```
#13 10.35 Traceback (most recent call last):
#13 10.35   File "aws", line 19, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscli/clidriver.py", line 20, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscli/botocore/session.py", line 27, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscli/botocore/client.py", line 17, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscli/botocore/args.py", line 26, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscli/botocore/config.py", line 16, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscli/botocore/endpoint.py", line 25, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscli/botocore/httpchecksum.py", line 27, in <module>
#13 10.35   File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
#13 10.35   File "awscrt/checksums.py", line 4, in <module>
#13 10.35 ImportError: Error relocating /aws/dist/_awscrt.cpython-39-x86_64-linux-gnu.so: pthread_attr_setaffinity_np: symbol not found
#13 10.35 [19] Failed to execute script aws
#13 11.11 You can now run: /usr/local/bin/aws --version
#13 DONE 11.6s
```

And if you try to run `aws` command:

```
$ docker run -ti --rm -e awscli=v2 k8s aws --version
Traceback (most recent call last):
  File "aws", line 19, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscli/clidriver.py", line 20, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscli/botocore/session.py", line 27, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscli/botocore/client.py", line 17, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscli/botocore/args.py", line 26, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscli/botocore/config.py", line 16, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscli/botocore/endpoint.py", line 25, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscli/botocore/httpchecksum.py", line 27, in <module>
  File "PyInstaller/loader/pyimod03_importers.py", line 531, in exec_module
  File "awscrt/checksums.py", line 4, in <module>
ImportError: Error relocating /usr/local/aws-cli-v2/v2/dist/_awscrt.cpython-39-x
86_64-linux-gnu.so: pthread_attr_setaffinity_np: symbol not found
[8] Failed to execute script aws
```

In order to use the latest version (which is built with glibc) the only way I found is to compile glibc for Alpine or to use a third-party package like mentioned here https://stackoverflow.com/a/61268529/6599436

Reply:
If you agree to use a third-party solution or would like to include glibc compilation process (which is a quite time consuming) into this Dockerfile — let me now